make.cmd

```diff
@@ -22,7 +22,14 @@ if errorlevel 1 (
 goto :end
 
 :build
+    go-winres make --product-version=%CLC_VERSION% --file-version=%CLC_VERSION%
+    go generate ./...

```

Comment:
Is this step necessary?

Reply:
I guess? Since it adds the Windows releated stuff.